gtsam/nonlinear/NonlinearFactorGraph.h

```diff
@@ -202,6 +203,29 @@ namespace gtsam {
       push_back(boost::make_shared<ExpressionFactor<T> >(R, z, h));
     }
 
+    /**
+     * Directly add a PriorFactor to the factor graph.
```

Comment:
Add a note mentioning the convenience such as 

```
Convenience function to add PriorFactors specially.

Reply:
Done 👍